lib/src/table/table-th/table-th.component.html

```diff
@@ -0,0 +1 @@
+<small class="font-weight-bolder" [class.text-muted]="!active">{{ label }}</small>
```

Comment:
this part a bit complicated to accept, first small tag will define a font-size that is smaller than the base, and font-weight-bolder will define a bolder weight (700 i think) and it will have !important since bootstrap does it like that.

This will create complications for whoever wants to use this components with custom styles.

Most of SEB usecases would align with this, since the header is small and bold but some wouldn't, in this case we should care about the edge cases

Reply:
Yes you're right, after checking the designs of my project and SEB Bootstrap I noticed this design is actually coming from my project and is not in Design Library/bootstrap. I will remove the whole thing and just use span with default SEB styles.